script/release/publish-to-npm.js

```diff
@@ -103,6 +103,29 @@ new Promise((resolve, reject) => {
 
     return release;
   })
+  .then(async (release) => {
+    const checksumsAsset = release.assets.find((asset) => asset.name === 'SHASUMS256.txt');
+    if (!checksumsAsset) {
+      throw new Error(`cannot find SHASUMS256.txt from v${rootPackageJson.version} release assets`);
+    }
+
+    const checksumsContent = await getAssetContents(
+      rootPackageJson.version.indexOf('nightly') > 0 ? 'nightlies' : 'electron',
+      checksumsAsset.id
+    );
+
+    const checksumsObject = checksumsContent.trim().split('\n').reduce((checksums, line) => {
+      const [checksum, file] = line.split(' *');
+      return {
+        [file]: checksum,
+        ...checksums
+      };
+    }, {});
```

Comment:
`reduce` is obfuscation here, this reads more simply and is fewer lines to boot.
```suggestion
    const checksumsObject = {};
    checksumsContent.trim().split('\n').forEach((line) => {
      const [checksum, file] = line.split(' *');
      checksumsObject[file] = checksum;
    });
```